src/main/java/Backend/Axes.java

```diff
@@ -18,12 +24,23 @@
 
 public class Axes implements Serializable {
     private float scale;
+    private float dimensionSize;
```

Comment:
quickly add a getter for dimension. By no means urgent, but would be nice to have in the future.